vividus/build.gradle

```diff
@@ -37,7 +37,8 @@ tasks.register('systemTest', Test) {
 test {
     jvmArgs = [
             '--add-opens=java.base/java.util=ALL-UNNAMED',
-            '--add-opens=java.base/java.lang=ALL-UNNAMED'
+            '--add-opens=java.base/java.lang=ALL-UNNAMED',
```

Comment:
why is it needed?

Reply:
Unit tests from `vividus:test` didn't pass without this

Reply:
what is the error? (I'm interested in this particular flag)

Reply:
```
testNoMatchedSteps(StdOut)
java.lang.NoClassDefFoundError: Could not initialize class org.vividus.util.json.JsonPathUtils
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:375)
	at org.vividus.configuration.Vividus.init(Vividus.java:38)
	at org.vividus.runner.StepsCounter.main(StepsCounter.java:65)
	at org.vividus.runner.StepsCounterTests.testCounter(StepsCounterTests.java:194)
	at org.vividus.runner.StepsCounterTests.testNoMatchedSteps(StepsCounterTests.java:91)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.ExceptionInInitializerError [in thread "Test worker"]
	at org.vividus.util.json.JsonPathUtils.setDistinctPathFunction(JsonPathUtils.java:92)
	at org.vividus.util.json.JsonPathUtils.<clinit>(JsonPathUtils.java:47)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:375)
	at org.vividus.configuration.Vividus.init(Vividus.java:38)
	at org.vividus.configuration.VividusTests.shouldDrawBannerAfterLoggerConfigurations(VividusTests.java:52)
	... 3 more
```